src/main/java/atel/model/ScriptFuncLib.java

```diff
@@ -153,10 +153,10 @@ public static void initialize() {
         putFuncWithIdx(0x0083, new ScriptFunc("getCurrentEntranceRotation", "float", null, true));
         putFuncWithIdx(0x0084, new ScriptFunc("waitForText", "unknown", null, p("boxIndex", "int"), p(2)));
         putFuncWithIdx(0x0085, new ScriptFunc("setTouchRadius", "unknown", null, p("radius", "float"))); // noclip: setCollisionRadius()
-        putUnknownFunc(0x0086, 0);
+        putFuncWithIdx(0x0086, new ScriptFunc("CurrentLevel", "unknown", null, false)); // the ID of the actual map geometry file, multiple "map"s can share these
```

Comment:
up to you on naming here. To clarify, I believe what you have as "map" here is really a script file/event. Script files define which map file (level geometry, collision, etc) to load. Same thing for `Warp/TransitionToMap` (you can't just pick a level without a script/event). I don't actually remember if this gets used for anything, though.